### PR TITLE
CORDA-1999 Changed isRelevant to relevancyStatus.

### PR DESCRIPTION
**Pull request by roger3cev**
Does what it says on the tin.

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/services/vault/NodeVaultService.kt [hunk 1]
```diff
@@ -113,8 +113,9 @@ class NodeVaultService(
                 // For EVERY state to be committed to the vault, this checks whether it is spendable by the recording
                 // node. The behaviour is as follows:
                 //
-                // 1) All vault updates marked as RELEVANT will, of, course all have isRelevant = true.
-                // 2) For ALL_VISIBLE updates, those which are not relevant according to the relevancy rules will have isRelevant = false.
+                // 1) All vault updates marked as RELEVANT will, of, course all have relevancyStatus = true.
```
[the remark this answers is not in the document]
Comment (roger3cev) [reply node/src/main/kotlin/net/corda/node/services/vault/NodeVaultService.kt]: Ah.. in the DB, the relevancy flag is actually a boolean but this wording isn't very clear. I'll change it.

[the remark this answers is not in the document]
Comment (josecoll) [reply node/src/main/kotlin/net/corda/node/services/vault/NodeVaultService.kt]: Actually it's an INT in the db - the default type used for Enum type mappings ;-)

[the remark this answers is not in the document]
Comment (roger3cev) [reply node/src/main/kotlin/net/corda/node/services/vault/NodeVaultService.kt]: Ah this is true :)

**Comment by roger3cev** [comment]
Over to you @josecoll !

File: node/src/main/kotlin/net/corda/node/services/vault/NodeVaultService.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (roger3cev) [reply node/src/main/kotlin/net/corda/node/services/vault/NodeVaultService.kt]: Ah whoops forgot... Just pushed up the changes now.

**Pull request merged**